src/ingest_validation_tools/directory-schemas/imc3d.yaml

```diff
@@ -17,6 +17,12 @@
   required: True
   is_qa_qc: True
 
+-
+  pattern: 'mcd/panel\.csv'
+  description: 'CSV containing a list of channels and antibodies used for acquisition'
```

Comment:
How is this different from the antibodies TSV?

Reply:
@mccalluc I have no clue. this is a question for @cebriggs7135 and we can discuss it today in our meeting. all i know this file was meant to be in `/extras` and then decided to pull it a level up.

Reply:
@mccalluc about the lowercase it really depends to be honest. moving forward I can suggest it as a best practice.